src/daemon/input_linux.c

```diff
@@ -128,14 +128,22 @@ void os_inputclose(usbdevice* kb){
     kb->uinput_mouse = 0;
 }
 
-// Generate SYN reports to synchronize device
-static void isync(usbdevice* kb){
+// Generate SYN reports to synchronize keyboard device
+static void isync_kb(usbdevice* kb){
```

Comment:
I'm not sure it's a good idea to separate them when they essentially do the same thing. Perhaps instead of passing a usbdevice* you can pass an int fd to the function and keep only one copy of the code that does both.